sources/fr/chireads.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+
+import logging
+from lncrawl.core.crawler import Crawler
+from lncrawl.utils.cleaner import TextCleaner
+import requests
+from bs4 import BeautifulSoup
+
+logger = logging.getLogger(__name__)
+
+
+class MyCrawlerName(Crawler):
+    base_url = ["https://chireads.com/"]
+    has_manga = False
+    machine_translation = False
+
+    def initialize(self):
+        self.cleaner = TextCleaner()
+        pass
+
+    def search_novel(self, query):
+        query = query.lower().replace(" ", "+")
+
+        # Using self.get_soup() here throw an error, I don't know why.
+        response = requests.get("https://chireads.com/search?x=0&y=0&name=" + query)
+        soup = BeautifulSoup(response.text, "html.parser")
+
+        result = []
+        content = soup.find("div", {"id": "content"})
+
+        for novel in content.find_all("li"):
+            content = novel.find("a")
+
+            all_titles = content.get("title").split("|")
+
+            if len(all_titles) == 3:
+                info = (
+                    "English: " + all_titles[1] + "\nOriginal title: " + all_titles[2]
+                )
+            elif len(all_titles) == 2:
+                info = "Other title: " + all_titles[1]
+            else:
+                info = None
+
+            to_add = {"title": all_titles[0], "url": content.get("href")}
+
+            if info:
+                to_add["info"] = info
+
+            result.append(to_add)
+        logger.info("Found %d results", len(result), ": ", result)
```

Comment:
Forgot a logger here, I don't know how to remove it without closing the pull request sry